.github/workflows/lighthouse.yml

```
@@ -0,0 +1,32 @@
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build app
+        run: npm run build
+
+      - name: Run Lighthouse CI
+        uses: treosh/lighthouse-ci-action@v12
+        with:
+          configPath: ./lighthouserc.json
+          staticDistDir: ./dist
```

actionlint (1.7.12)

[error] input "staticDistDir" is not defined in action "treosh/lighthouse-ci-action@v12". available inputs are "artifactName", "basicAuthPassword", "basicAuthUsername", "budgetPath", "configPath", "runs", "serverBaseUrl", "serverToken", "temporaryPublicStorage", "uploadArtifacts", "uploadExtraArgs", "urls"

(action)